windows/bpf_load.c

```diff
@@ -27,5 +33,45 @@ load_bpf_file(const char* filename)
 
     bpf_object__close(object);
 
+    return 0;
+}
+
+int // 0 on success and sets prog_fd and map_fd, non-zero on failure and sets bpf_log_buf.
+load_xdp_root_bpf_file(const char* filename)
+{
+    struct bpf_object_open_opts opts = { 0 };
+    struct bpf_object* object = bpf_object__open_file(filename, &opts);
+    printf("Opened file %s\n",filename);
+    if (object == NULL) {
+        printf("Object is NULL");
+        return -1;
+    }
+    bpf_object__load(object);
+    struct bpf_program* program = bpf_object__next_program(object, NULL);
+
+    if (bpf_program__pin(program, program_path) < 0) {
```

Comment:
I had to pin the xdp root program to keep it loaded so added a separate function load_xdp_root_bpf_file